x-pack/legacy/plugins/canvas/i18n/ui.ts

```diff
@@ -628,6 +628,27 @@ export const ViewStrings = {
           markdown: MARKDOWN,
         },
       }),
+    getOpenLinksInNewTabDisplayName: () =>
+      i18n.translate('xpack.canvas.uis.views.openLinksInNewTabTitle', {
+        defaultMessage: 'Markdown link settings',
+        values: {
+          markdown: MARKDOWN,
+        },
+      }),
+    getOpenLinksInNewTabLabelName: () =>
+      i18n.translate('xpack.canvas.uis.views.openLinksInNewTabLabel', {
+        defaultMessage: 'Open all links in a new tab',
+        values: {
+          markdown: MARKDOWN,
+        },
+      }),
+    getOpenLinksInNewTabHelp: () =>
+      i18n.translate('xpack.canvas.uis.views.openLinksInNewTabHelpLabel', {
+        defaultMessage: 'Set toggle value to true to open all links in a new tab',
```

Comment:
Let's shorten this a bit so that it follow the text in the other arguments:
→ "Set links to open in new tab"

![image](https://user-images.githubusercontent.com/446285/74162267-0c59e280-4be6-11ea-9512-923ec8a635a4.png)

Reply:
@ryankeairns thanks, I've updated the message to "Set links to open in new tab"